src/styles/components/_buttons.scss

```diff
@@ -145,6 +145,36 @@ body.user-is-tabbing button:focus {
     }
   }
 }
+.ft-button--wrapper{
+  padding: 2px 2px 2px 2px;
+  background-image: linear-gradient(to right, #0018AB , #9933CC );
+  border-radius: 4px;
+  box-sizing: border-box;
+  display: block;
+  margin: 0 auto;
+  /*-webkit-border-image: -webkit-linear-gradient(to right, #0018AB , #9933CC ) 5;
```

Comment:
Did you intend to comment these out? Do these need to be deleted?

Reply:
I meant to comment them out, in case we would need to make this gradient button standard and not editable. Or at least that was my thinking. Would you like me to go in and delete it?